### PR TITLE
Max cap on map persistence

### PR DESCRIPTION
**Pull request by Zth--**
title

### CODE REVIEW

**Comment by DamianX** [comment]
Is this what was crashing the server on deff?

**Comment by Exxion** [comment]
~30000 ashes, yes

**Comment by Zth--** [comment]
Yup http://game.ss13.moe/uploads/2020-05-05_13-54-34.txt

File: code/modules/persistence/map_persistence_type.dm [hunk 1]
```diff
@@ -30,8 +31,12 @@
 	fdel(writing)
 
 //Note: We save all items. Even if they're in space etc. Next round will be in charge of seeing if they're valid. I don't expect any significant performance loss from this, but if so, this can be changed easily.
+//Note 2: The cap was added because reading 30k of objects crashes the server
 /datum/map_persistence_type/proc/writeSavefile()
 	var/list/finished_list = list()
+	if(tracking.len > arbitrary_max_objects)
+		log_debug("[time_stamp()] - Map persistence \"[name]\" hit the cap. [tracking.len - arbitrary_max_objects] objects did not make it.")
```
Comment (DamianX) [inline code/modules/persistence/map_persistence_type.dm]:
```suggestion
		log_debug("Map persistence \"[name]\" hit the cap. [tracking.len - arbitrary_max_objects] objects did not make it.")
```
log_debug already timestamps

**Pull request merged**